app/modules/entities/components/editor/entity_edit.svelte

```diff
@@ -42,13 +45,15 @@
     <EntityEditMenu {entity} />
   </div>
 
-  {#if !hasMonolingualTitle}
-    <LabelsEditor {entity} bind:favoriteLabel />
+  {#if hasMonolingualTitle === false}
+    <LabelsEditor bind:entity bind:favoriteLabel />
   {/if}
 
-  {#each Object.entries(typePropertiesPerCategory) as [ category, categoryProperties ]}
-    <PropertyCategory {entity} {category} {categoryProperties} />
-  {/each}
+  {#if typePropertiesPerCategory}
+    {#each Object.entries(typePropertiesPerCategory) as [ category, categoryProperties ]}
```

Comment:
When editing [this entity](https://inventaire.io/entity/inv:00f118336b02219e1bddc8fa930e64be/edit), when i select P31 Comics, then change first author to inker, author is duplicated in Original Language field
![2023-11-18_08-07](https://github.com/inventaire/inventaire-client/assets/5363918/b42f8ad6-0d79-4736-9a0e-023c5f6ff015)

Reply:
Should have been fixed by 228941f0e